examples/google-auth/README.md

```diff
@@ -7,27 +7,25 @@
   Gatsby Functions Google Auth Starter
 </h1>
 
-
 ## 🚀 Quick start
 
 1.  **Set up Google OAuth Client.**
-    
+
     This sample uses google oAuth client to authenticate into the Google inbox api. Obtain an oAuth credential from the Google Cloud [API & Services Credentails Page](https://console.cloud.google.com/apis/credentials?)
+
     - Generate an Oauth Client ID. You can follow the steps highlighted in [Account Authentication - Service Account](https://theoephraim.github.io/node-google-spreadsheet/#/getting-started/authentication?id=oauth). Make sure to set up at least one **Authorized redirect URI** to be `http://localhost:8000`
     - Enable the [Gmail API](https://console.cloud.google.com/apis/api/gmail.googleapis.com) in your Google Cloud project.
 
-
 2.  **Add the Google Oauth Credentials to your `.env.development` file**
 
     There are **2** environment variable you'll need to add to your project:
 
     - `GOOGLE_CLIENT_ID`: Fetch the ClientId from the newly created OAuth 2.0 Client ID. Yuo can find it in the [Google Credentials Console](https://console.cloud.google.com/apis/credentials)
     - `GOOGLE_CLIENT_SECRET`: Fetch the Client Secret from the newly created OAuth 2.0 Client ID. Yuo can find it in the [Google Credentials Console](https://console.cloud.google.com/apis/credentials)
 
-    You'll also want to add these as environment variables when deploying to Gatsby Cloud. 
-
+    You'll also want to add these as environment variables when deploying to Gatsby Cloud.
 
-2.  **Start developing.**
+3.  **Start developing.**
```

Comment:
I noticed a readme above was converted to 1. for all of the steps? Was that intended and do we want to change all the readme's?

Reply:
I don't think so. Lemme check locally.

Reply:
fixed. Not sure why prettier did that.